insta_reels_publishing_api_sample/index.js

```diff
@@ -193,8 +220,14 @@ app.post("/uploadReels", async function (req, res) {
     if(typeof locationId === 'undefined') {
         locationId = "";
     }
-    const uploadParamsString = `caption=${caption}&cover_url=${coverUrl}&thumb_offset=${thumbOffset}&location_id=${locationId}&access_token=${req.session.userToken}`;
-    const uploadVideoUri = `https://graph.facebook.com/v14.0/${accountId}/media?media_type=REELS&video_url=${videoUrl}&${uploadParamsString}`;
+    const uploadVideoUri = buildGraphAPIURL(`${accountId}/media`, {
+        media_type: 'REELS',
+        video_url: videoUrl,
+        caption,
```

Comment:
We don't have to write this as `caption: caption,`? Is it implied?

Reply:
Yeah, it is implied. If the property name has the same name as the variable you can use the [shorthand notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions).

Reply:
Oh that's convenient! Thanks for explaining!